src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java

```diff
@@ -189,4 +188,14 @@ private String getSubscriptionDetails(String newSubscription, String oldSubscrip
         }
         return subscription;
     }
+
+    private void overrideNotificationType(NotificationWrapper notificationWrapper) {
+        SscsCaseData newSscsData = notificationWrapper.getNewSscsCaseData();
+        SscsCaseData oldSscsData = notificationWrapper.getOldSscsCaseData();
+        if (CASE_UPDATED.equals(notificationWrapper.getNotificationType())
+                && (null == oldSscsData || null == oldSscsData.getCaseReference() || oldSscsData.getCaseReference().isEmpty())
```

Comment:
Create a new method to make this more readable (line 196 + 197), ```hasCaseReferenceBeenAdded```.  I've always planned to do that, just never got around to doing it, sorry.